### ISSUE TITLE
user.username, Commit To It 

### ISSUE THREAD

**Issue by ga7pilot** [jlord/git-it#192]
After:
$ git config --global user.username ga7pilot

I get:

ACTUAL                             EXPECTED
"Seems there are changes"       != "Changes have been committed!" 
"to commit still."              != ""                             
""                              != null                           
# FAIL

Your solution to COMMIT TO IT didn't match the expected output.
Try again!

My github account is new and was set up just for this tutorial. The email was verified and I am able to log in and view my github profile. Tried various combinations of names and quotes. Very confused. 

Browsed other issues without finding a resolution. On someone else's issue you asked for the following:

$ git config user.username
ga7pilot
$ git config user.name
Anthony Gutsch
$ git config user.email
ga7pilot@comcast.net

Any ideas? Thanks.

**Comment by ga7pilot** [comment]
It's resolved. Wish I could say how. Started over from the challenge. Must have missed something.

**Comment by ga7pilot** [comment]
After passing once on "COMMIT TO IT", I continued to the next excise and it has failed again back on "COMMIT TO IT". Any ideas?

When I:
$ git-it verify

I get:
Verifying "COMMIT TO IT"...

ACTUAL                                    EXPECTED
------                                            --------
"Seems there are changes"       != "Changes have been committed!" 
"to commit still."                          != ""                             
""                                                 != null                           
FAIL

Your solution to COMMIT TO IT didn't match the expected output.
Try again!

**Comment by jlord** [comment]
Are you saying that you're running verify on the challenge _after_ Commit to It and it's still running the Commit to It verify rather than the GitHubbin' verify?

**Comment by ga7pilot** [comment]
Yes, that is what I am saying. I have since gone on to complete your entire workshop and it is still running the commit to it verify.  I figured out that it will now pass commit to it verify only if I am in the hello-world directory, even though it initially it failed in that directory, but it won't quit running the commit to it verify. I think it is confused now because I went on.

I don't think verify was really that necessary anyway. The workshop was well organized enough that I was able to understand what was going on throughout the workshop, and achieve the learning objectives without verifying. Just a little confusion about the user.username vs user.name thing.

Since I completed the workshop, we can close this, unless you have anymore questions.
